github/ci/prow-deploy/files/jobs/kubevirt/kubevirtci/kubevirtci-presubmits.yaml

```diff
@@ -529,6 +529,34 @@ presubmits:
           privileged: true
       nodeSelector:
         type: bare-metal-external
+  - always_run: true
```

Comment:
Let's start with `always_run: false` and `optional: true`.

Reply:
np, it was still draft ;]